.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+maxColumn = 112
```

Comment:
> I suggest that we go for the gold maxcolumn standard, 80. Having it larger than that hinders readability in other devices like mobile phones or laptops.

80 column comes from IBM punch cards - https://en.wikipedia.org/wiki/Punched_card
I often code from my laptop, but never thought "hmmm, I wish the code base had 80 column max." Changing our code base so someone can read our code in mobile phones make no sense. Is the advent of the mobile phones the reason why we are switching to Scalafmt?

Why not 132 columns (number of characters printable by IBM 1403), or 100 columns ([Google's Java Guide](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit))?